Panto/css/style.css

```diff
@@ -26,9 +26,13 @@
   gap: 6.4rem;
 }
 
+.main-nav-list:first-child a {
```

Comment:
don't use selectors like this. instead use classes to handle this.
what if the design changes and you have to move this link to the second position? your style would break.